tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
I'm failing to see the tests for copy and move semantics. Am I just not seeing them or was this missed? Inside `TestHelpers.hpp` there is a function that tests these. You can look at `DataVector` for an example of how to test.